WhoIsThatServer.Recognition/Recognition/RecognitionServices.cs

```diff
@@ -133,5 +133,28 @@ public async Task<bool> InsertPersonInToGroup(ImageModel imageModel)
 
             return true;
         }
+
+        /// <summary>
+        /// Detects features of face
+        /// </summary>
+        /// <param name="imageModel">Object which is sent from frontend</param>
+        /// <returns>Age, gender of face and whether glasses and facialhair are present</returns>
+        public async Task<Face[]> DetectFeaturesOfFace(ImageModel imageModel)
+        {
+            try
+            {
+                var faces = await _faceServiceClient.DetectAsync(RecUtil.GetStreamFromUri(imageModel.ImageContentUri), false, true, new FaceAttributeType[] {
```

Comment:
That method looks clunky a bit... and two booleans as parameters make me think twice

Reply:
Also you should move stream creation to a separate line

Reply:
First boolean is for whether this method should return ID of face and second one is whether I request face features. This wrapper is sketchy anyway, so it might look interesting

Reply:
Wouldnt it make sense to make 2 methods instead?

Reply:
Also, you could have default value as null for this second bool, and provide options only when you use this feature

Reply:
I got to dig deeper into this method, it's a bit confusing for me. As for making it in two methods, not sure what you exactly had in mind, but I wouldn't know how to split it in two methods and I don't need the ID it returns. Then again, I have to invest more time in that